packages/services/f-feature-management/src/vue/store/features.module.js

```diff
@@ -0,0 +1,21 @@
+import SET_FM_CONFIG_JSON from './mutation.types';
+
+export default {
+    state: () => ({
+        configJson: {}
+    }),
+
+    actions: {
+        getConfigJson: ({ state }) => state.configJson,
```

Comment:
Not sure this is necessary, you should just be able to read `configJson` directly via `mapState`.